research-software-directory/RSD-as-a-service
# 200 project url table

# Migrate project links to separate table

Fixes #200 

Changes proposed in this pull request:
* The project links are moved from project table to separate table. Legacy data is migrated      

How to test:
* `docker-compose down --volumes`: remove old instance
* `docker-compose build`: build new images
* `docker-compose up`: start new version
*  navigate to projects and confirm that project links are present, compare with legacy pages. The `fixme` and `todo` non-existing links should not be in the new RSD version

PR Checklist:
